File: AppBox3/Apps/Pedometer/Model/Pedometer+CoreDataProperties.m
//
//  Pedometer+CoreDataProperties.m
//  AppBox3
//
//  Created by Byeong Kwon Kwak on 5/12/16.
//  Copyright © 2016 ALLABOUTAPPS. All rights reserved.
//
//  Choose "Create NSManagedObject Subclass…" from the Core Data editor menu
//  to delete and recreate this implementation file for your updated model.
//

#import "Pedometer+CoreDataProperties.h"

@implementation Pedometer (CoreDataProperties)

@dynamic uniqueID;
@dynamic date;
@dynamic numberOfSteps;
@dynamic distance;
@dynamic floorsAscended;
@dynamic floorsDescended;

@end

File: AppBox3/Apps/Pedometer/Model/Pedometer_+CoreDataProperties.swift
//
//  Pedometer_+CoreDataProperties.swift
//  
//
//  Created by BYEONG KWON KWAK on 10/2/24.
//
//

import Foundation
import CoreData


extension Pedometer_ {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Pedometer_> {
        return NSFetchRequest<Pedometer_>(entityName: "Pedometer_")
    }

    @NSManaged public var date: String?
    @NSManaged public var distance: NSNumber?
    @NSManaged public var floorsAscended: NSNumber?
    @NSManaged public var floorsDescended: NSNumber?
    @NSManaged public var numberOfSteps: NSNumber?
    @NSManaged public var uniqueID: String?

}
